### DISCUSSION TITLE
Is there a way to enable clicking through the transparent taskbar? (windows 11 taskbar styler)

### DISCUSSION THREAD

**Discussion by Looodon** [ramensoftware/windhawk-mods#884] [Q&A]
Hi,
I picked the DockLike Theme from the windows 11 taskbar plug-in and was wondering if it's possible to be able to click through the areas that are transparent? startallback has that feature so i thought i'd ask 
Thank you!

**Comment by m417z** [comment]
Right now it's not possible, see: https://github.com/ramensoftware/windhawk-mods/issues/805
